source/includes/collector-api/_actions-api.md

```diff
@@ -192,6 +192,66 @@ moesif.track('Clicked Sign Up', {
 });
 ```
 
+```javascript--nodejs
+var moesifapi = require('moesifapi');
+var apiClient = moesifapi.ApiController;
+
+moesifapi.configuration.ApplicationId = "YOUR_COLLECTOR_APPLICATION_ID";
+
+// Define the request context objects for each action.
+var req_contextA = {
+  time: new Date(),
+  uri: "https://api.acmeinc.com/items/reviews/",
+  ipAddress: "61.48.220.123",
+  userAgentString: "Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:47.0) Gecko/20100101 Firefox/47.0"
+};
+
+var req_contextB = {
+  time: new Date(),
+  uri: "https://api.acmeinc.com/pricing/",
+  ipAddress: "61.48.220.126",
+  userAgentString: "PostmanRuntime/7.26.5"
+};
+
+// Define the actions.
+var actionA = {
+  transactionId: "a3765025-46ec-45dd-bc83-b136a8d1d357",
+  actionName: "Clicked Sign Up",
+  sessionToken: "23abf0owekfmcn4u3qypxg09w4d8ayrcdx8nu2ng]s98y18cx98q3yhwmnhcfx43f",
+  userId: "18340",
+  companyId: "25100",
+  metadata: {
+    email: "alex@acmeinc.com",
+    button_label: 'Get Started',
+    sign_up_method: 'Google SSO'
+  },
+  request: req_contextA
+};
+
+var actionB = {
+  transactionId: "a3765024-46ee-45dd-bc83-b136c8d1d250",
+  actionName: "Viewed pricing",
+  sessionToken: "23jdf0owejfmbn4u3qypxg09w4d8ayrxdx8nu2ng]s98y18cx98q3yhwmnhcfx43f",
+  userId: "12390",
+  companyId: "97895",
+  metadata: {
+    email: "kim@acmeinc.com",
+    button_label: 'See pricing',
+    sign_up_method: 'Google SSO'
+  },
+  request: req_contextB
+};
+
+var actions = [
+  new moesifapi.ActionModel(actionA),
+  new moesifapi.ActionModel(actionB)
+];
+// Send the batch of Actions
+apiClient.sendActionsBatch(actions, function(error, response, context) {
+  // Do Something
+});
+```
+
 |Name|Type|Required|Description|
 |-----------|-----------|-----------|-----------|
```

Comment:
any reason for using table for type description?
I have seen that this format looks broken in [nuget package repository](https://www.nuget.org/packages/Moesif.Middleware)